src-electron/main-process/electron-main.js

```diff
@@ -107,8 +107,10 @@ function createWindow () {
   })
 
   mainWindow.webContents.on('will-navigate', (e, url) => {
-    e.preventDefault()
-    shell.openExternal(url)
+    if (url !== e.sender.getURL()) {
```

Comment:
Not sure what this does exactly?

Reply:
This basically overloads hyperlink navigation and instead of opening the new link in Electron, it opens it in an external browser. The unintended consequence of this is the navigation associated with reloading the app would open in the browser. This fixes this by catching that behavior.